itest/rust/src/register_tests/res/registered_docs.xml

```diff
@@ -17,38 +17,54 @@ public class Player : Node2D
   code&amp;nbsp;block
 &lt;/div&gt;[/codeblock][br][br][url=https://www.google.com/search?q=2+%2B+2+&lt;+5]Google: 2 + 2 &lt; 5[/url][br][br]connect these[br][br]¹ [url=https://example.org]https://example.org[/url][br]² because the glorb doesn&#39;t flibble.[br]³ This is the third footnote in order of definition.[br]⁴ Fourth footnote in order of definition.[br]⁵ This is the fifth footnote in order of definition.[br]⁶ sixth footnote in order of definition.</description>
 <methods>
-<method name="ye">
+<method name="ye"  >
   <return type="f32" />
   
   <description>
   huh
   </description>
 </method>
 
-<method name="process_node">
+<method name="process_node"  >
   <return type="Gd &lt; Node &gt;" />
   <param index="0" name="node" type="Gd &lt; Node &gt;" />
   <description>
   Function with lots of special characters ([code]Gd&lt;Node&gt;[/code])
   </description>
 </method>
 
-<method name="_virtual_documented">
+<method name="_virtual_documented"  >
   <return type="()" />
   <param index="0" name="node" type="Gd &lt; Node &gt;" />
   <description>
   some virtual function that should be overridden by a user[br][br]some multiline doc[br][br]The [code]Gd&lt;Node&gt;[/code] param should be properly escaped
   </description>
 </method>
 
-<method name="ne">
+<method name="ne"  >
```

Comment:
Do you know why these whitespace changes occur at the end?

Reply:
Good catch, I've been adding newlines even when tags were not present.

I decided to move whitespace into XML attribute itself, not sure if it is not too silly :thinking:. I'm fine with current form though